### PR TITLE
add eq10q: LV2 EQ plugins and more, with 64 bit processing

### PR DESCRIPTION
**Pull request by magnetophon**

### CODE REVIEW

**Comment by iyzsong** [comment]
Look good to me.
cc @cillianderoiste

File: pkgs/applications/audio/eq10q/default.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+  fixupPhase = ''
+    cp -r $out/var/empty/local/lib $out
+    rm -R $out/var
+  '';
+
+  meta = {
+    description = "LV2 EQ plugins and more, with 64 bit processing";
+    longDescription = ''
+      Up to 10-Bands parametric equalizer with mono and stereo versions
```
Comment (jagajaga) [inline pkgs/applications/audio/eq10q/default.nix]: Use dots in the `longDescription` :)

**Comment by jagajaga** [comment]
What about travis?

File: pkgs/applications/audio/eq10q/default.nix (hunk 1 above)
Comment (magnetophon) [inline pkgs/applications/audio/eq10q/default.nix]: @jagajaga done! :)

**Comment by lethalman** [comment]
Built locally, thanks.

**Pull request merged**